wdl/TasksMakeCohortVcf.wdl

```diff
@@ -990,6 +990,7 @@ task ScatterVcf {
 
   command <<<
     set -euo pipefail
+    ~{if !defined(vcf_index) then "tabix ~{vcf}" else ""}
```

Comment:
This is problematic if the vcf is unsorted. Can you make an explicit input like `Boolean generate_index_if_unavailable` that defaults to false but is also required to be true here?

Reply:
Removing this line and requiring index in AnnotateVcf as discussed